docs/platform/concepts/service_backups.rst

```diff
@@ -167,6 +167,14 @@ Aiven for ClickHouse backups contain database lists, table schemas, table conten
 
     For more information on Aiven for ClickHouse backups, see :ref:`Backup and restore <backup-and-restore>`.
 
+Where do we store the backups?
+------------------------------
+
```

Comment:
Please move the whole paragraph and the screenshot to section "About backups at Aiven". Include them after sentence "All Aiven services, except for Apache Kafka® and M3 Aggregator/Coordinator, have time-based backups that are encrypted and securely stored."